lacework/resource_lacework_query.go

```diff
@@ -0,0 +1,166 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/lacework/go-sdk/api"
+	"github.com/pkg/errors"
+)
+
+func resourceLaceworkQuery() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkQueryCreate,
+		Read:   resourceLaceworkQueryRead,
+		Update: resourceLaceworkQueryUpdate,
+		Delete: resourceLaceworkQueryDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkQuery,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"query_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The id of the query",
+			},
+			"query": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The query string",
+			},
+			"evaluator_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The query evaluator id",
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					return old == "<<IMPLICIT>>" || new == "<<IMPLICIT>>"
```

Comment:
I'm not sure that after this we could change the evaluator ID but I also think those are edge cases where the user wants to repurpose a query, so let us ignore this for now. 😅